src/tools/dartpad/index.md

```diff
@@ -136,3 +136,22 @@ for (var char in 'hello'.split('')) {
 The language features and APIs that DartPad supports depend on the
 **Dart SDK** version that DartPad is based on.
 You can find the SDK version at the bottom right of DartPad.
+
+## Embedding DartPad in web pages {#embedding}
+
+You can embed DartPad inside of web pages,
+customizing it to suit your purpose.
+For example, the [futures codelab][]
+contains multiple embedded DartPads
+labeled as _examples_ and _exercises_.
+
+For more information about how to use embedded DartPads, see
+[best practices for using DartPad in tutorials][].
+
+[futures codelab]: /codelabs/async-await
```

Comment:
Nit: move this to the end of the page along with the other named link definitions?